recipes/glfw/all/conanfile.py

```diff
@@ -129,7 +129,7 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "glfw"
         self.cpp_info.names["cmake_find_package_multi"] = "glfw"
         self.cpp_info.names["pkg_config"] = "glfw3"
-        self.cpp_info.builddirs = [self._module_subfolder]
+        self.cpp_info.builddirs.append(self._module_subfolder)
```

Comment:
Could you please explain to me, what is the default value for `self.cpp_info.builddirs`?
Why do we append here, but for `self.cpp_info.build_modules` we use assignment?

Reply:
It's explained here: https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info

Reply:
Thanks 👍